drape/drape_tests/glyph_mng_tests.cpp

```diff
@@ -89,8 +89,12 @@ class GlyphRenderer
 
 UNIT_TEST(GlyphLoadingTest)
 {
-  // This unit test creates window so can't be run in GUI-less Linux machine.
-#ifndef OMIM_OS_LINUX
+#ifdef OMIM_OS_LINUX
+  // This unit test creates a window so can't be run in GUI-less Linux machine.
+  if (nullptr == ::getenv("DISPLAY"))
```

Comment:
Isn't it's possible to just set `QT_QPA_PLATFORM=qoffscreen`?

Reply:
If the mentioned window is created by Qt, set this environment variable before running tests and it will pick Qt's offscreen platform plugin that does not create any gui hence does not require DISPLAY.

Reply:
Thanks, good idea. Changed the code.